src/main/java/com/ericsson/ec/subscriptionobject/SubscriptionObject.java

```diff
@@ -0,0 +1,87 @@
+package com.ericsson.ec.subscriptionobject;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import lombok.Getter;
+
+abstract class SubscriptionObject {
+    private static final String SUBSCRIPTION_TEMPLATE_PATH = "src/main/resources/subscriptionsTemplate.json";
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Getter
+    protected ObjectNode subscriptionJson;
+
+    /**
+     * Creates a subscription with a specific name.
+     * @param subscriptionName
+     * @throws IOException
+     */
+    public SubscriptionObject(String subscriptionName) throws IOException {
+        URL subscriptionsInput = new File(SUBSCRIPTION_TEMPLATE_PATH).toURI().toURL();
```

Comment:
Can't the objectmapper just read this as a File instead of URL? Seems unnecessary to convert it so many times.

Reply:
Oh ye, that works :)